src/getDataset.js

```diff
@@ -133,7 +137,7 @@ const getDataset = async (req, res) => {
   }
 
   const baseUrl = req.url.split(query.prefix)[0];
-  let redirectUrl = baseUrl + '/' + auspiceDisplayUrl;
+  let redirectUrl = path.join(baseUrl, auspiceDisplayUrl);
   if (query.type) {
     redirectUrl += `&type=${query.type}`;
   }
```

Comment:
`path.join()` will vary across operating systems.  This could be addressed by using `require("path/posix")` or `path.posix.join()`, but that said, the semantics of filesystem paths aren't identical to those of URL paths, despite being very similar.

Looking more broadly at this chunk of code, the existing form is already problematic for several reasons, including that whether your other query params get passed along or not with the baseUrl depends on where they are relative to the prefix param. (Also, while unlikely in this case, if your prefix value appears elsewhere in the URL earlier than the query param, it all breaks.) A better approach is to replace the `prefix` query value in the request URL with the desired new value using URL handling libs instead of string manip. Something like:

    const redirectUrl = new URL(req.url);
    redirectUrl.searchParams.set("prefix", auspiceDisplayUrl);